File: loops/fib.js


const fib = () => {
    let result = [0, 1];

    while (result.length < 11) {
        let temp = result[result.length - 2] + result[result.length - 1];
        result.push(temp);
    }

    console.log("Fibonacci sequence:");
    console.log(result.join(' '));
}


fib();

File: loops/fib.py


def fib():
    result = [0, 1]

    while len(result) < 11:
        temp = result[-2] + result[-1]
        result.append(temp)

    s = map(lambda x: str(x), result)

    print('Fibonacci sequence:')
    print(" ".join(s))

fib()